### PR TITLE
Update Set-CsTenantFederationConfiguration.md

### PR DESCRIPTION
**Pull request by luis-msftpm**
Adding information regarding -AllowTeamsConsumerInbound and -AllowTeamsConsumer.
This new properties were introduced as part of the effort to enable communications between people using Teams with an account managed by an organization and people using Teams with an account that's not managed by an organization.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Comment by dariomws** [comment]
Close https://github.com/MicrosoftDocs/office-docs-powershell/issues/8581.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md [hunk 1]
```diff
@@ -176,6 +183,33 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -AllowTeamsConsumer
+Allows federation with people using Teams with an account that's not managed by an organization.
+```yaml
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md:188]:
```suggestion

```yaml
```

File: skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md [hunk 2]
```diff
@@ -176,6 +183,33 @@ Default value: None
...
+Required: False
+Position: Named
+Default value: True
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowTeamsConsumerInbound
+Allows people using Teams with an account that's not managed by an organization, to discover and start communication with users in your organization. When -AllowTeamsConsumer and this is disabled, only the users in your organization will be able to discover and start communication with people using Teams with an account that's not managed by an organization, but they will not discover and start communications with users in your organization.
+```yaml
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md:202]:
```suggestion

```yaml
```

**Comment by yogkumgit** [comment]
@dariomws

**Comment by dariomws** [comment]
@luis-msftpm, could you review suggestions? Thanks.

**Comment by luis-msftpm** [comment]
> @luis-msftpm, could you review suggestions? Thanks.

@dariomws Done. Thx.

**Comment by yogkumgit** [comment]
@dariomws

**Comment by yogkumgit** [comment]
@dariomws Please review.

**Comment by luis-msftpm** [comment]
@dariomws - This PR has been open for over a month. We have companies opening ICMs due to this document being outdated. Can you please prioritize this? Is there anyone else that can help approve and merge this?

**Comment by dariomws** [comment]
@luis-msftpm, I think you are still missing a suggestion.

"When -AllowTeamsConsumer and this is disabled"

Should be as per my understanding:

"When -AllowTeamsConsumer is enabled and this parameter is disabled"

**Comment by luis-msftpm** [comment]
> @luis-msftpm, I think you are still missing a suggestion.
> 
> "When -AllowTeamsConsumer and this is disabled"
> 
> Should be as per my understanding:
> 
> "When -AllowTeamsConsumer is enabled and this parameter is disabled"

Fixed now. Thanks for catching that one.

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md [hunk 3]
```diff
@@ -120,6 +120,13 @@ Example 6 shows how you can add domains to the Allowed Domains using a List coll
 First, a List collection is created and domains are added to it, then, simply include the AllowedDomainsAsAList parameter and set the parameter value to the List object.
 When this command completes, the allowed domain list will be filled with those domains.
 
+### -------------------------- Example 7 --------------------------
+```
+Set-CsTenantFederationConfiguration -AllowTeamsConsumer $True -AllowTeamsConsumerInbound $False
+```
+
+The command shown in Example 1 enables communication with people using Teams with an account that's not managed by an organization, to only be initiated by people in your organization. This means that people using Teams with an account that's not managed by an organization will not be able to discover or start a conversation with people in your organization.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md:128]:
```suggestion
The command shown in Example 7 enables communication with people using Teams with an account that's not managed by an organization, to only be initiated by people in your organization. This means that people using Teams with an account that's not managed by an organization will not be able to discover or start a conversation with people in your organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @luis-msftpm, kindly commit the additional suggested change. Many thanks.

**Comment by luis-msftpm** [comment]
> Copy review - @luis-msftpm, kindly commit the additional suggested change. Many thanks.

@opbld33 - Done

**Comment by luis-msftpm** [comment]
@opbld33 - can you please help get this closed and merged today? We have customers asking for this doc to be updated since we started rolling out over a month ago. Would really appreciate the help here.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**